packages/braid-design-system/src/lib/components/PageBlock/PageBlock.gallery.tsx

```diff
@@ -5,6 +5,15 @@ import { PageBlock } from '../';
 import source from '@braid-design-system/source.macro';
 
 export const galleryItems: ComponentExample[] = [
+  {
+    label: 'Small width',
+    Example: () =>
+      source(
+        <PageBlock width="small">
+          <Placeholder height={100} />
+        </PageBlock>,
+      ),
+  },
```

Comment:
Can't really see the different sizes here because the gallery frame is too narrow

Reply:
Yeah its the same for `ContentBlock` too. I wonder if we should consider removing them from Gallery entirely. They arent exactly a visual pattern people browse for in that way

Reply:
Mm yep could do. Unless people like to use Gallery to grab code snippets for these components, but I'm not sure that's happening

Reply:
We'll do a Gallery sweep separately. For now we'll proceed with following the pattern in there.